src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1452,7 +1454,8 @@ because xbuild doesn't support framework reference assemblies.
       FrameworkDirectories="$(_XATargetFrameworkDirectories);$(_XATargetFrameworkDirectories)Facades"
       AcwMapFile="$(_AcwMapFile)"
       SupportedAbis="@(_BuildTargetAbis)"
-      SkipJniAddNativeMethodRegistrationAttributeScan="$(_SkipJniAddNativeMethodRegistrationAttributeScan)">
+      SkipJniAddNativeMethodRegistrationAttributeScan="$(_SkipJniAddNativeMethodRegistrationAttributeScan)"
+      CheckedBuild="$(AndroidCheckedBuild)">
```

Comment:
`$(AndroidCheckedBuild)` should be documented within `Documentation/guides/building-apps/build-properties.md`.

Reply:
It was there, but we decided we're not shipping the checked builds - does it make sense to document the property?

Reply:
Can we make it?
```xml
<_AndroidCheckedBuild Condition=" '$(_AndroidCheckedBuild)' == '' ">false</_AndroidCheckedBuild>
```